event_pubsub/producers/blockchain_event_producer.py

```diff
@@ -43,9 +43,16 @@ def _produce_contract_events(self, start_block_number, end_block_number, net_id)
 
     def _get_end_block_number(self, last_processed_block_number, batch_limit):
         current_block_number = self._blockchain_util.get_current_block_no()
+        logger.info(f"Current block number={current_block_number}")
+
```

Comment:
This is a global change correct? This will impact all consumers?

Reply:
yes, it will impact, which means at wort case currently, it is 2 min now it is going to be 3 min, sir.